### ISSUE TITLE
Merge README.rst and README.template.rst

### ISSUE THREAD

**Issue by rchristie** [OpenCMISS-Examples/template_example#6]
I strongly suggest the contents of the README.template.rst simply be added to the end of the README.rst file, with an introductory sentence saying to edit it to remove the instructions and fill in the template below.

It's needless work to have to rename and delete the file, and we've already had developers committing both.

**Comment by hsorby** [comment]
I don't think it is overly onerous to delete and rename the template file, I think this is not significantly different from deleting a chunk of pre-text.
